src/App.jsx

```diff
@@ -14,33 +18,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_ALPHABET = 'alphabet';
+const SORT_LENGTH = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, { sortField, isReverse }) {
+  const preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      if (sortField === 'alphabet') {
+        return good1.localeCompare(good2);
+      }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+      if (sortField === 'length') {
+        return good1.length - good2.length;
+      }
+
+      return preparedGoods;
```

Comment:
The return statement inside the sort function should return a number, not the `preparedGoods` array. This line should be removed or replaced with a valid comparison result. The `sort` function expects a return value of -1, 0, or 1 to determine the order of the elements.